src/api-guides/payment-provider-integration.html

```diff
@@ -0,0 +1,109 @@
+
+* **Training Toolkit** at [http://www.concurtraining.com/](http://www.concurtraining.com/) - These resources help you learn the Concur product basics, administration and reporting.
+* **Shared Technical Services Videos** at [http://www.concurtraining.com/prdeployment/sts](http://www.concurtraining.com/prdeployment/sts) - The videos found here will help you understand the unique types of transactions that can occur in the Expense product and their resulting impacts on the financial picture in the extract file.
+
+## <a name="integration"></a>Integration
+
+### <a name="quick-connect"></a>Quick Connect
+
+[Quick Connect Scope for Enterprise Apps](./quick-connect-scope-for-enterprise-apps.html)
+
+**Quick Connect** describes the process customers use to connect their Concur site with an App Center Partner's Enterprise application. See the separate Quick Connect scope document for details to guide you through the development of this required piece to your certified application.
+
+### <a name="profile"></a>Company Profile
+
+[Profile Company API v1](/api-reference/profile-beta/company.html#get)
+
+### <a name="Topics-to-complete-prior-to-obtaining-invoices"></a>Topics to complete prior to obtaining invoices
+
+**These topics should be completed prior to obtaining Invoices pending payment:**
+
+#### Topic 1: Update the field called, "Payment Method Type" within the SAP-Concur Invoice Vendor record
+
+In order to pay a customer's invoices, the SAP-Concur Invoice Vendor record must have the value of **PAYPVD** populated within the "Payment Method Type" field.  This can be populated in 3 ways, one of which is via API.  The Partner and customer need to agree who will be responsible for updating the Vendors initially and on an on-going basis. Here are the methods the Vendor (aka Supplier) record can be updated:
+
+Option|Technique|Owner responsible for update
+---|---|---
+Option 1|File import-On-Demand Import via an Excel template within the SAP-Concur Invoice product|Customer completes
+Option 2|File import-Scheduled Import via a csv file|Customer completes
+Option 3|App integration|Customer and Partner need to coordinate since Vendor master data most likely resides in their ERP
+
+### <a name="Update-Vendor-Payment-Method-field"></a>Update Vendor Payment Method field
+
+[Vendor Update API v3](/api-reference/invoice/v3.vendor.html#put)
+
+#### Topic 2: Customer ensures the Vendor Contact Email addresses exist within the SAP-Concur Invoice Vendor record for the Vendors that the Partner will pay
+
+Confirm that the customer adminsitrator has created a Concur Audit Rule to require the Vendor (Supplier) email address has been populated for those Vendor invoices that will be paid via this integration. The Customer can take the following steps to create the Audit Rule:
+
+#### Audit Rule step 1:
+Navigate to the Administration menu, then Invoice, then Audit Rules.
+Click New
+
+Name: Vendor Supplier Contact Email
```

Comment:
You will need to make these a bulleted list or a table because single spacing is interpreted as a single line.

Reply:
Same for all of the audit rules.